clang/include/clang/Basic/Attr.td

```diff
@@ -1328,6 +1328,15 @@ def SYCLDeviceHas : InheritableAttr {
   let SupportsNonconformingLambdaSyntax = 1;
 }
 
+def SYCLDeviceCompileOptions : InheritableAttr {
```

Comment:
I don't see this attribute being used anywhere, why is it needed?

Reply:
I revisited the desig and yes, this is not required. Thanks